.gitignore

```diff
@@ -85,3 +85,5 @@ dist
 
 # Ignore when people install with Yarn
 yarn.lock
+/.vscode
+/.vs
```

Comment:
Do you mind reverting this commit? VS Code is the recommended editor and I think it's helpful to provide a standard configuration to complement the setup in the [Development Setup section of the Contributing Guide](https://github.com/lansingcodes/www/blob/master/.github/CONTRIBUTING.md#development-setup). 😉

Reply:
Sure think! VS code just added a number of files that I didn't want to push up since no one before me had so I ignored the file assuming it was my personal settings.

Reply:
Huh, I'm curious about those files but let's save that for later.

If it happens again, I think you can run `git update-index –assume-unchanged` to skip those changes in your next commit without having to update `.gitignore`. Alternatively, feel free to submit a PR with the files/changes and we can scrutinize them together. 👍